configurablecare/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
remove this file? gitignore is already there in fhir-app-examples at root level.

Reply:
I guess we still need this to mention the files to be specifically ignored within this directory

Reply:
Oh then we should change root level gitignore to ignore al build files. Anyway this is NIT.